File: scripts/Subscriber.py
import rospy
from performance_tests.msg import SuperAwesome

def callback(data):
    rospy.loginfo("%s", data.data)
 
def listener():
    rospy.init_node('Subscriber_Python', anonymous=True)
    rospy.Subscriber("SuperAwesome_Topic", SuperAwesome, callback)
    rospy.spin()

if __name__ == '__main__':
    listener()

File: src/Subscriber.cpp
#include "ros/ros.h"
#include "std_msgs/String.h"
#include "performance_tests/SuperAwesome.h"

void SuperAwesomeCallback(const performance_tests::SuperAwesome::ConstPtr& msg)
{
    ROS_INFO("%s", msg->data.c_str());
}

int main(int argc, char **argv)
{
    ros::init(argc, argv, "Subscriber_Cpp");
    ros::NodeHandle n;   
    ros::Subscriber SuperAwesome_Subscriber = n.subscribe("SuperAwesome_Topic", 1000, SuperAwesomeCallback);
    ros::spin();
    
    return 0;
}